org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/BaseInitHandler.java

```diff
@@ -103,6 +111,15 @@ public InitializeResult initialize(InitializeParams param) {
 			Preferences prefs = Preferences.createFrom((Map<String, Object>) settings);
 			prefs.setRootPaths(rootPaths);
 			preferenceManager.update(prefs);
+			if (!firstTimeWorkspaceIsOpened) {
+				// We don't care about triggering a full build here, like in onDidChangeConfiguration
+				try {
+					JVMConfigurator.configureJVMs(prefs);
+					firstTimeWorkspaceIsOpened = true;
```

Comment:
nope. We need the very 1st time the "Workspace" is opened/initialized, i.e. when .metadata is created

Reply:
Fixed.